examples/tests/backend_driver.py

```diff
@@ -205,7 +205,7 @@
     'multiline.py',
     'multiple_figs_demo.py',
     'nan_test.py',
-    'newscalarformatter_demo.py',
+    'plot_scalarformatter.py',
```

Comment:
If I understood properly from a conversation with @tacaswell, this file is way out of date and unmaintained.
@tacaswell can you confirm or correct this?